pkg/urbit/daemon/main.c

```diff
@@ -446,6 +446,7 @@ u3_ve_usage(c3_i argc, c3_c** argv)
     "-d            Daemon mode; implies -t\n",
     "-e url        Ethereum gateway\n",
     "-F ship       Fake keys; also disables networking\n",
+    "-G            Boot using key-string from key file\n",
```

Comment:
Sweet! "key-string from key file" is a bit confusing though. Do we pass the string or the file?

Might be more appropriately worded as "Private key string", to mirror the "Private key file" from `-k`. Also want to make clear you pass something in. ie:

```suggestion
    "-G 0wEtc      Private key string\n",
```

Arguably this should specify that the string is a `@uw`-encoded jamfile, but probably not worth the complexity in this context.

Reply:
Let me know if the changes I just committed make sense to address the concerns you have above. Very good points, all around.